web/packages/teleport/src/Roles/RoleEditor/StandardEditor/StandardEditor.test.tsx

```diff
@@ -159,6 +161,24 @@ test('edits resource access', async () => {
   expect(role.spec.allow.logins).toEqual(['{{internal.logins}}', 'ec2-user']);
 });
 
+test('propagates version info to Kubernetes resource access', async () => {
```

Comment:
maybe rename this test to "triggers v6 validation for kubernetes resource."

I initially got confused otherwise since the test only says propagates version info but we are interested in validation triggered at the end.